### DISCUSSION TITLE
Error when download Power Pages code using VS Code

### DISCUSSION THREAD

**Discussion by haryadihart** [microsoft/powerplatform-build-tools#651] [Q&A] [answered]
Hi All,

I got this issue when using Power Platform CLI to download Power Pages code.
how do i resolve this issue?
syntax i used is : pac paportal download --path --websiteID

![image](https://github.com/microsoft/powerplatform-build-tools/assets/152245015/a0eafea6-22b3-4d0d-841f-1f877885cc11)

Thanks

**Comment by safwankarim-sentient** [comment, marked as answer]
I had the same error in windows. Its something wrong with the verison of the pac cli. I found a work around by deleteing the files pac cli files inside here `%USERPROFILE%\.dotnet\tools` and then installing an older version which i know works, `dotnet tool install --global Microsoft.PowerApps.CLI.Tool --version 1.24.3`
If you have installed through vscode, you can uninstall it there and use the dotnet code to install it. 
One downside to this is you have to manually update the pac cli version in the future.

**Reply by haryadihart** [reply]
Hi, I have tried you suggestion. 
It works like a charm.
Thank you.

do you know maybe when should I get updates for latest version later? where can i check the update?

**Reply by safwankarim-sentient** [reply]
Glad that worked mate!
Just remove the `--version` flag and it will download the latest pac cli to your machine.
